File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/LaundryPreferences.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Andrew on 3/6/2018.
 */

public class LaundryPreferences {
    @SerializedName("rooms")
    @Expose
    private List<Integer> rooms;

    public LaundryPreferences(List<Integer> rooms) {
        this.rooms = rooms;
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/laundry/classes/LaundryPreferences.kt
package com.pennapps.labs.pennmobile.laundry.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LaundryPreferences {
    @SerializedName("rooms")
    @Expose
    val rooms: List<Int>? = null
}
